tcwebhooks-core/src/main/java/webhook/teamcity/settings/converter/OldStyleBuildState.java

```diff
@@ -10,6 +10,7 @@ private OldStyleBuildState(){}
     public static final Integer BEFORE_BUILD_FINISHED 	= Integer.parseInt("00001000",2);
     public static final Integer RESPONSIBILITY_CHANGED 	= Integer.parseInt("00010000",2);
     public static final Integer BUILD_INTERRUPTED 		= Integer.parseInt("00100000",2);
+	public static final Integer BUILD_TAGGED 			= Integer.parseInt("01000000",2);

```

Comment:
We don't need to change this file. It's only used for translating the old, bit based, build states to the "new" format (from about 10 years ago).